koku/masu/processor/orchestrator.py

```diff
@@ -93,14 +93,23 @@ def get_accounts(billing_source=None, provider_uuid=None):
                     all_accounts = [account]
 
         for account in all_accounts:
-            if disable_cloud_source_processing(account.get("schema_name")) and not provider_uuid:
-                LOG.info(f"Cloud source processing disabled for {account.get('schema_name')}")
+            schema_name = account.get("schema_name")
+            if disable_cloud_source_processing(schema_name) and not provider_uuid:
+                LOG.info(log_json("get_accounts", "processing disabled for schema", schema=schema_name))
                 continue
             if disable_source(provider_uuid):
+                LOG.info(
+                    log_json(
+                        "get_accounts",
```

Comment:
Is this the correct tracing ID?

Reply:
There is no tracing id in this function, so I just used the function name.